lib/get-xo.js

```diff
@@ -0,0 +1,19 @@
+/** @babel */
+import {allowUnsafeNewFunction} from 'loophole';
+import getPackagePath from './get-package-path';
+
+// (base: string) => xo: Promise<Object>
+export default function getXO(base) {
+	const getResolvePath = () => base ? getPackagePath(base) : Promise.resolve();
+
+	return getResolvePath()
```

Comment:
Why make this async? It's only a few fs calls, and the sync version would be much faster.